addons/common/XEH_postInit.sqf

```diff
@@ -85,78 +85,74 @@ if (isServer) then {
+["ace_playMove", {(_this select 0) playMove (_this select 1)}] call CBA_fnc_addEventHandler;
+["ace_playMoveNow", {(_this select 0) playMoveNow (_this select 1)}] call CBA_fnc_addEventHandler;
+["ace_switchMove", {(_this select 0) switchMove (_this select 1)}] call CBA_fnc_addEventHandler;
+["ace_setVectorDirAndUp", {(_this select 0) setVectorDirAndUp (_this select 1)}] call CBA_fnc_addEventHandler;
+["ace_setVanillaHitPointDamage", {(_this select 0) setHitPointDamage (_this select 1)}] call CBA_fnc_addEventHandler;
 
 // Request framework
-[QGVAR(requestCallback), FUNC(requestCallback)] call FUNC(addEventHandler);
-[QGVAR(receiveRequest), FUNC(receiveRequest)] call FUNC(addEventHandler);
+[QGVAR(requestCallback), FUNC(requestCallback)] call CBA_fnc_addEventHandler;
+[QGVAR(receiveRequest), FUNC(receiveRequest)] call CBA_fnc_addEventHandler;
 
-[QGVAR(systemChatGlobal), {systemChat _this}] call FUNC(addEventHandler);
+[QGVAR(systemChatGlobal), {systemChat _this}] call CBA_fnc_addEventHandler;
 
 if (isServer) then {
-    ["hideObjectGlobal", {(_this select 0) hideObjectGlobal (_this select 1)}] call FUNC(addEventHandler);
-    ["enableSimulationGlobal", {(_this select 0) enableSimulationGlobal (_this select 1)}] call FUNC(addEventHandler);
-    ["setOwner", {(_this select 0) setOwner (_this select 1)}] call FUNC(addEventHandler);
-    [QGVAR(serverLog), FUNC(serverLog)] call FUNC(addEventHandler);
+    [QGVAR(hideObjectGlobal), {(_this select 0) hideObjectGlobal (_this select 1)}] call CBA_fnc_addEventHandler;
+    [QGVAR(enableSimulationGlobal), {(_this select 0) enableSimulationGlobal (_this select 1)}] call CBA_fnc_addEventHandler;
+    ["ace_setOwner", {(_this select 0) setOwner (_this select 1)}] call CBA_fnc_addEventHandler;
+    [QGVAR(serverLog), FUNC(serverLog)] call CBA_fnc_addEventHandler;
 };
 
 
 //////////////////////////////////////////////////
 // Set up remote execution
 //////////////////////////////////////////////////
 
-// ACE events
-"ACEg" addPublicVariableEventHandler {_this call FUNC(_handleNetEvent)};
-"ACEc" addPublicVariableEventHandler {_this call FUNC(_handleNetEvent)};
-
 // Synced ACE events
 // Handle JIP scenario
 if (!isServer) then {
-    ["PlayerJip", {
+    ["ace_playerJIP", {
         ACE_LOGINFO("JIP event synchronization initialized");
-        ["SEH_all", [player]] call FUNC(serverEvent);
-    }] call FUNC(addEventHandler);
+        [QGVAR(SEH_all), [player]] call CBA_fnc_serverEvent;
+    }] call CBA_fnc_addEventHandler;
 } else {
-    ["SEH_all", FUNC(_handleRequestAllSyncedEvents)] call FUNC(addEventHandler);
+    [QGVAR(SEH_all), FUNC(_handleRequestAllSyncedEvents)] call CBA_fnc_addEventHandler;
 };
 
-["SEH", FUNC(_handleSyncedEvent)] call FUNC(addEventHandler);
-["SEH_s", FUNC(_handleRequestSyncedEvent)] call FUNC(addEventHandler);
+[QGVAR(SEH), FUNC(_handleSyncedEvent)] call CBA_fnc_addEventHandler;
+[QGVAR(SEH_s), FUNC(_handleRequestSyncedEvent)] call CBA_fnc_addEventHandler;
```

Comment:
This name was chosen to minimize network traffic. I suggest using a as-short-as-possible name- 4 letters max.
E.g.: https://github.com/CBATeam/CBA_A3/blob/master/addons/events/script_component.hpp#L20